popde/density_estimate.py

```diff
@@ -0,0 +1,92 @@
+# First script for iterative KDE
+from awkde import GaussianKDE  #we will need our own code 
+
+
+class  densityestimator(object):
```

Comment:
See https://realpython.com/python-pep8/#naming-styles 
The name of the class should use CamelCase
Also, if this is supposed to be a class that does _adaptive_ KDE with cross validation optimization, the name should reflect that
As I wrote in the development plan, we should have a much more basic density estimator class which _just_ calculates a KDE without any other functionality, what you have written is not that, so please give this a suitable name